src/allmydata/test/data/node.pem.txt

```diff
@@ -0,0 +1,45 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Ah, good question.  I don't think so because the Tub *doesn't* validate the certificate when it loads it (apart from making sure the private key matches, I think) and this code never uses this Tub to set up a TLS connection.  However, it does sort of make the cert a trap that might catch someone up if they change the test to do TLS or use the cert for something else in the test suite.

You can create a Tub without a certificate but if you do then it generates one itself.  My intuition is that generating certificates is slow ... but that might be wrong/outdated.  I see that instantiating a Tub without a cert on my laptop takes 28ms.  That is slow for making an object but it's not *slow* slow ... so maybe I should just let it do that - at least once.

Reply:
I tried running `time tox -e py27 -- allmydata.test.test_storage_client` against master and this PR branch a few times, and could not find any perceivable difference in running time.